### DISCUSSION TITLE
On Integral Logs it only Navigates but does not gather?

### DISCUSSION THREAD

**Discussion by miseryvein** [FFXIV-CombatReborn/GatherBuddyReborn#211] [General]
I am going to try to see if it happens with a few other things im trying to gather but so far it will do the gift of the land for crystals then move to next node, and again will not touch the logs

**Comment by miseryvein** [comment]
moved to my next thing, and working as intended? so what ius causing this with the logs only? ive been using this for crystal gathering and alumen gathering and now i tried those logs and it just gave up

**Comment by miseryvein** [comment]
seems to be a lot of things actualy, but not sure what causes it

**Comment by Recursive-Entity-Matrix** [comment]
Finish your BTN job quests. GBR is trying to use BTN skills you haven't unlocked and isn't able to.
